Dockerfile

```
@@ -0,0 +1,22 @@
+FROM python:3.9
```

Could we use the `-slim` image here?
```suggestion
FROM python:3.9-slim
```